Billing.API.Test/InvoiceApiTest.cs

```diff
@@ -64,7 +64,7 @@ public async Task GetInvoices_WhenToken_ReturnsResponse(HttpStatusCode httpStatu
             appFactory.Server.PreserveExecutionContext = true;
             var client = appFactory.CreateClient();
 
-            var request = new HttpRequestMessage(HttpMethod.Get, $"https://localhost/invoices/{clientId}");
+            var request = new HttpRequestMessage(HttpMethod.Get, $"https://localhost/accounts/0/{clientId}/invoices");
```

Comment:
What do you think about using a realistic origin (doppler/relay):

```suggestion
            var request = new HttpRequestMessage(HttpMethod.Get, $"https://localhost/accounts/doppler/{clientId}/invoices");
```